apps/events/models.py

```diff
@@ -388,19 +391,28 @@ def was_created_by_system(self) -> bool:
 
     def as_string(self, as_timezone: str | None = None):
         header_str = f"{self.name} (Message id={self.external_id}, message={self.prompt_text})"
-        if self.repetitions == 0:
+        if not self.repetitions or self.repetitions <= 1:
```

Comment:
Will repetitions ever be negative? I think we can just do the `if not self.repetitions` check?

Reply:
I don't think it will be negative. The issue with just checking 'not repetitions' is that a repetition of `1` is still a once off message since it only fires once.

Reply:
Right yes :+1: